api/build.gradle

```diff
@@ -20,23 +20,23 @@ buildscript {
     // External properties on the default project. Values declared in ext blocks
     // outside of the buildscript block aren't usable here.
     ext {
-        GAE_VERSION = '2.0.26'
+        GAE_VERSION = '2.0.29'
         GOOGLE_TRUTH_VERSION = '1.1.3'
-        GSON_VERSION = '2.9.1'
-        HIBERNATE_VERSION = '6.4.4.Final'
-        JACKSON_VERSION = '2.16.1'
-        LIQUIBASE_VERSION = '4.27.0'
-        LOGBACK_VERSION = '1.5.5'
+        GSON_VERSION = '2.10'
+        HIBERNATE_VERSION = '6.5.2.Final'
+        JACKSON_VERSION = '2.17.2'
+        LIQUIBASE_VERSION = '4.29.0'
+        LOGBACK_VERSION = '1.5.6'   // upgrading past this results in an error
```

Comment:
a few notes like this - I plan to return to these and re-attempt, or at least add more detail